stdlib/array.md

```diff
@@ -0,0 +1,1008 @@
+---
+title: Array
+---
+
+Utilities for working with arrays.
+
+<details>
+<summary>Added in <code>0.2.0</code></summary>
+<table>
+<thead>
+<tr><th>version</th><th>changes</th></tr>
+</thead>
+<tbody>
+<tr><td><code>0.1.0</code></td><td>Originally named `arrays`</td></tr>
+<tr><td><code>0.2.0</code></td><td>Renamed to `array`</td></tr>
```

Comment:
So, there's technically edge-case bug with these: markdown inside html isn't processed as markdown. I'm not sure the best way to handle (I really don't want to parse and convert it in graindoc).

Reply:
Deferring to @ospencer on what the best option would be here

Reply:
I talked with @ospencer and he said we should just leave this for now.